src/components/UriViewer.js

```diff
@@ -56,9 +56,11 @@ const isFilePreviewable = ({ size, ...metadata }) => {
   return !isBinary(metadata) && (isText(metadata) || (isImage(metadata) && size <= 1e9))
 }
 
-const isGs = uri => _.startsWith('gs://', uri)
+export const isGs = uri => _.startsWith('gs://', uri)
 
-const isDrs = uri => _.startsWith('dos://', uri) || _.startsWith('drs://', uri)
+export const isDrs = uri => _.startsWith('dos://', uri) || _.startsWith('drs://', uri)
+
+export const isAzureUri = uri => _.includes('blob.core.windows.net', uri)
```

Comment:
probably not a big deal, but this is a lot looser of a check than the `startsWith` checks for gs, dos, drs. This would match "ftp://somehost/somefile?downloadedFrom=blob.core.windows.net", for instance.

Reply:
Agreed, this is a tough one to predict/be more specific as the URIs start with some unique identifier, rather than a URL prefix -- e.g. `sa226344b664da26ad6863` is the prefix to a sample Azure URI

Reply:
Is there a regex we could use for this? Something along the lines of `https://(.+).blob.core.windows.net`?

Reply:
Updated!